include/onnxruntime/core/graph/graph.h

```diff
@@ -368,8 +369,21 @@ class Node {
                           to ensure the complete Graph is valid.
   */
   void ToProto(ONNX_NAMESPACE::NodeProto& proto, bool update_subgraphs = false) const;
+
+  Status SaveToOrtFormat(flatbuffers::FlatBufferBuilder& builder,
+                         flatbuffers::Offset<onnxruntime::experimental::fbs::Node>& fbs_node) const;
+
+  flatbuffers::Offset<onnxruntime::experimental::fbs::NodeEdge>
+  SaveEdgesOrtFormat(flatbuffers::FlatBufferBuilder& builder) const;
+
```

Comment:
nit: SaveEdgesToOrtFormat for consistency. Or remove the 'To' and 'From' everywhere as it's implied by Save and Load.